### PR TITLE
Shm11 kyrtaq

### PR DESCRIPTION
**Pull request by kyrtaq**

### CODE REVIEW

File: shm/player.cpp [hunk 1]
```diff
@@ -6,3 +6,11 @@ Player::Player(const Ship& ship,
     : ship_(std::make_shared<Ship>(ship)),
       money_(money),
       availableSpace_(availableSpace) {}
+
+size_t Player::calcAvailableSpace() {
+    availableSpace_ = 0;
+    for (const auto el : ship_->getCargos()) {
+        availableSpace_ += el->getAmount();
+    }
+    return availableSpace_;
```
Comment (NG90) [inline shm/player.cpp:15]: This version of function returns whole  amount of cargos on ship, not availableSpace. You should substract this value from ship->capacity_ in order to get available space on ship.

File: shm/ship.hpp [hunk 2]
```diff
@@ -22,6 +22,7 @@ class Ship {
     size_t getId() const { return id_; }
     size_t getCrew() const { return crew_; }
     Cargo* getCargo(size_t index) { return cargo_[index].get(); }
+    std::vector<std::shared_ptr<Cargo>> getCargos() { return cargo_; }
```
Comment (NG90) [inline shm/ship.hpp:25]: add const to this getters, like above e.g. `getCrew()`

File: shm/player.cpp (hunk 1 above)
Comment (kyrtaq) [reply shm/player.cpp:16]: done

File: shm/ship.hpp (hunk 2 above)
Comment (kyrtaq) [reply shm/ship.hpp]: done

**Pull request merged**